### PR TITLE
Convert all api calls to .format()

### PR DESCRIPTION
**Pull request by noqqe**
as recommended in #6 :)

### CODE REVIEW

File: crowd_api/__init__.py [hunk 1]
```diff
@@ -10,247 +10,251 @@
...
+            return {"status": False, "code": req.status_code, "reason": req.content}
+
+    def get_group_users(self, **kwargs):
+        users = []
+
+        if "groupname" not in kwargs:
+            raise ValueError("Must pass username")
+
+        req = self.api_get("/group/user/direct?groupname={}&max-results={}".format(
+            kwargs['groupname'], str(kwargs.get('max_results', 1000))))
```
Comment (m4ce) [inline crowd_api/__init__.py]: no need to cast to str, try it out

**Review by m4ce — CHANGES_REQUESTED** [review]

File: crowd_api/__init__.py (hunk 1 above)
Comment (noqqe) [reply crowd_api/__init__.py]: You are right :) Done

**Comment by noqqe** [comment]
Are you okay with the pep8 changes?

**Comment by m4ce** [comment]
> Are you okay with the pep8 changes?

Yes, that's fine

**Comment by m4ce** [comment]
Thanks for the contribution!

**Pull request merged**